src/backend/gitporter/update_all_workspaces.py

```diff
@@ -1,10 +1,11 @@
 from database import createLocalSession, models as dbm
-from .update_workspace import update_workspace
+from .update_session_repositories import update_session_repositories
+
 
 def update_all_workspaces():
     r"""
     TODO: ignore workspaces with sessions older than 30 days
     """
-    with createLocalSession() as session:
-        for workspace in session.query(dbm.Workspace):
-            update_workspace(workspace_name=workspace.name, platform=workspace.platform)
+    with createLocalSession() as connection:
+        for session in connection.query(dbm.Session).all():
```

Comment:
meinen neuen `last_seen` bauen wir in einem späteren PR ein!?

Reply:
last_seen war noch nicht da als ich das gebaut habe. später ein kleiner PR klingt super